nibabel/minc2.py

```diff
@@ -77,7 +77,7 @@ def _get_dimensions(self, var):
             dimorder = var.attrs['dimorder'].decode()
         except KeyError:  # No specified dimensions
             return []
-        return dimorder.split(',')
+        return dimorder.split(',')[:len(var.shape)]
```

Comment:
I'll check in a test case. Some MINC files contain stray "dimorder" attributes that are longer than the variable itself, e.g. a file may have a scalar image-min with a dimorder=zspace attribute.